hat/assets/js/apps/Iaso/domains/registry/index.tsx

```diff
@@ -0,0 +1,121 @@
+        '& ul[role="tree"]': {
+            maxHeight: '75vh',
+            '& li>div>div>div svg': {
+                display: 'none',
+            },
+        },
+    },
+}));
+
+type Params = {
+    accountId: 'string';
+};
+
+type Props = {
+    params: Params;
+};
+
+export const Registry: FunctionComponent<Props> = ({ params }) => {
+    const classes: Record<string, string> = useStyles();
+    const { formatMessage } = useSafeIntl();
+
+    const [selectedOrgUnits, setSelectedOrgUnits] = useState<OrgUnit[]>([]);
+
+    const [selectedOrgUnitsIds, setSelectedOrgUnitsIds] = useState<string[]>(
+        [],
+    );
+    const [selectedOrgUnitParents, setSelectedOrgUnitParents] = useState<
+        Map<string, string>
+    >(new Map());
+    const onUpdate = (orgUnitIds, parentsData, orgUnits) => {
+        setSelectedOrgUnitsIds(orgUnitIds);
+        setSelectedOrgUnitParents(parentsData);
+        if (orgUnits) {
+            setSelectedOrgUnits(orgUnits);
+        }
+    };
+    return (
+        <>
+            <TopBar
+                title={formatMessage(MESSAGES.title)}
+                displayBackButton={false}
+            />
+            <Box className={classes.containerFullHeightNoTabPadded}>
+                <Grid container spacing={2}>
+                    <Grid item xs={12} md={6} className={classes.treeContainer}>
+                        <TreeViewWithSearch
+                            getChildrenData={id => getChildrenData(id, 'VALID')}
+                            getRootData={(id, type) =>
+                                getRootData(id, type, 'VALID')
+                            }
+                            label={makeTreeviewLabel(classes, true)}
+                            toggleOnLabelClick={false}
+                            request={(value, count) => {
+                                //  @ts-ignore => TODO fix this while migrate tree to Typescript
+                                return searchOrgUnits({ value, count });
+                            }}
+                            makeDropDownText={orgUnit => (
+                                <OrgUnitLabel
+                                    orgUnit={orgUnit}
+                                    withType
+                                    withSource={false}
+                                />
+                            )}
+                            toolTip={tooltip}
+                            parseNodeIds={getOrgUnitAncestors}
+                            preselected={selectedOrgUnitsIds}
+                            preexpanded={selectedOrgUnitParents}
+                            selectedData={selectedOrgUnits}
+                            onUpdate={onUpdate}
+                            multiselect={false}
+                            inputLabelObject={MESSAGES.search}
+                        />
+                    </Grid>
+                    <Grid item xs={12} md={6}>
+                        {selectedOrgUnits?.length > 0 && (
```

Comment:
I think it would look less empty if we showed the Paper with  placeholder values when there's no org unit selected yet. Or alternately have an animation/transition i.o the Paper just popping up, which doesn't give a great effect

Reply:
Yes the paper is not on the mock-up, let's wait for customer / Martin feed-back first